Standalone Demo/Standalone Demo/AppDelegate.swift

```diff
@@ -17,15 +17,19 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         HyBid.setTestMode(true)
 // Step 5: Set Location Tracking (Optional)
         HyBid.setLocationTracking(true)
-// Step 6: Set Interstitial skipOffet (Optional)
-        HyBid.setInterstitialSkipOffset(2)
-// Step 7: Set Targetting (Optional)
+// Step 6: Set HTML Interstitial skipOffet (Optional)
+        HyBid.setHTMLInterstitialSkipOffset(2)
+// Step 7: Set Video Interstitial skipOffet (Optional)
+        HyBid.setVideoInterstitialSkipOffset(5)
+// Step 8: Set Custom Click Behaviour (Optional)
+        HyBid.setInterstitialActionBehaviour(HB_CREATIVE)
+// Step 9: Set Targetting (Optional)
```

Comment:
The word "Targeting" is spelled with one T. I've fixed it in the other sample apps. Can you fix here as well? 👍  _(The original comment here has a typo)_